src/timepicker/TimePickerColumnState.ts

```diff
@@ -1,46 +1,25 @@
-import React from "react";
 import { useCompositeState } from "reakit";
+import { useControllableState } from "@chakra-ui/hooks";
 
-interface Props {
-  onChange?: (v: number | string) => void;
-  onSelection?: (v: number | string) => void;
-  value?: number | string;
+interface TimePickerColumnInitialProps {
+  value?: number;
+  onChange?: (v: number) => void;
```

Comment:
Why are we only accepting number here? 🤔 what happened to "AM" and "PM" meridians

Reply:
This happened 😛 

0 for "AM"  |    1 for "PM"

What do you think ?
 
![image](https://user-images.githubusercontent.com/39694575/95460164-3987ac00-0992-11eb-891a-f6674a52d414.png)

Reply:
I'm fine with it, as long as the Column still accepts and works with arbitrary values